docs/basic-features/script.md

```diff
@@ -7,6 +7,7 @@ description: Next.js helps you optimize loading third-party scripts with the bui
 <details>
   <summary><b>Examples</b></summary>
   <ul>
+    <li><a href="https://github.com/vercel/next.js/tree/canary/examples/script-component">Script Component</a></li>
```

Comment:
I like your idea of putting together all the different strategies into a single example! I may do the same and consolidate all the other examples into a separate, single example focused on showing different third-party providers (example: `script-component-recipes`)

Reply:
Thanks! If you need any help with that just let me know 🙌